scripts/infra/nvidia-setup.sh

```diff
@@ -113,15 +113,15 @@ fi \
     && cp -a /etc/dnf/dnf.conf{,.tmp} && mv /etc/dnf/dnf.conf{.tmp,} \
     && dnf config-manager --best --nodocs --setopt=install_weak_deps=False --save \
     && dnf config-manager --add-repo https://developer.download.nvidia.com/compute/cuda/repos/rhel${OS_VERSION_MAJOR}/${CUDA_REPO_ARCH}/cuda-rhel${OS_VERSION_MAJOR}.repo \
-    && dnf -y module enable nvidia-driver:${DRIVER_STREAM}/default \
+    && dnf -y module enable nvidia-driver:${DRIVER_STREAM}-open/default \
```

Comment:
This seems like it could have performance implications and a diversion from downstream(s).  Do we not at least need the option to use the non open version?

Reply:
@fabiendupont what's the version used in downstream? Is it 570? Which steam is (to be) used there?

Reply:
I remove my concern.  It wasn't clear to me from the comments that downstream had switched to -open with 570 as well.  Apologies for the confusion.